File: webservice/Frikollection_Api/Frikollection_Api/DTOs/Collection/FollowedCollectionDto.cs
using Frikollection_Api.DTOs.Product;

namespace Frikollection_Api.DTOs.Collection
{
    public class FollowedCollectionDto
    {
        public Guid CollectionId { get; set; }
        public string Name { get; set; }
        public Boolean? Private { get; set; }
        public string? OwnerNickname { get; set; }
        public DateTime? FollowDate { get; set; }
        public ICollection<ProductDto> Products { get; set; } = new List<ProductDto>();
    }
}

File: mobile_desktop_app/FrikollectionMobileDesktop/composeApp/src/commonMain/kotlin/org/example/frikollection_mobile_desktop/models/collection/FollowedCollectionDto.kt
package org.example.frikollection_mobile_desktop.models.collection

import kotlinx.serialization.Serializable
import org.example.frikollection_mobile_desktop.models.product.ProductDto

@Serializable
data class FollowedCollectionDto(
    val collectionId: String,
    val name: String,
    val private: Boolean,
    val ownerNickname: String,
    val followDate: String,
    val products: List<ProductDto>
)